docs/source/cli_help/github.rst

```diff
@@ -8,7 +8,7 @@ github
 
     git machete github <subcommand>
 
-where ``<subcommand>`` is one of: ``anno-prs``, ``create-pr``, ``retarget-pr``.
+where ``<subcommand>`` is one of: ``anno-prs``, ``checkout-pr``, ``create-pr``, ``retarget-pr``.
```

Comment:
Yeah... so... last minute change to be considered... as per #231, it might make more sense to use `checkout-prs` (plural) name for this subcommand 🤔